src/shared-components/selectorButton/index.js

```diff
@@ -41,38 +44,52 @@ const SelectorButton = ({
   icon,
   size,
   ...rest
-}) => (
-  <OuterContainer
-    onClick={onClick}
-    onKeyPress={onClick}
-    tabIndex="0"
-    selector={selector}
-    {...rest}
-  >
-    <SelectorContainer>
-      <SelectorIcon>
-        {checked ?
-          <CheckmarkIcon
-            css={css`
-              color: ${COLORS.white};
-            `}
-            width={16}
-            height={16}
-          /> :
-          size === 'large' && icon
-        }
-      </SelectorIcon>
-      <Selector type={type} checked={checked} selector={selector} size={size}/>
-    </SelectorContainer>
+}) => {
+  const checkedIcon =
+    selector === 'radio' ? (
+      <CircleSolidIcon
+        css={css`
+          color: ${COLORS.white};
+        `}
+        width={8}
+        height={8}
+      />
+    ) : (
+      <CheckmarkIcon
+        css={css`
+          color: ${COLORS.white};
+        `}
+        width={16}
+        height={16}
+      />
+    );
+  return (
+    <OuterContainer
+      onClick={onClick}
+      onKeyPress={onClick}
+      tabIndex="0"
```

Comment:
This may be out of scope, but it might be a good idea to add a `role` of `checkbox` and an `aria-checked` attribute to help assistive technologies determine the state of the selector. 

```jsx
role={selector} // 'radio' or 'checkbox'
aria-checked={checked}
```

I'm very much a beginner when it comes to accessibility, so definitely possible I'm missing other stuff that would be helpful too.